### ISSUE TITLE
Using the system language.

### ISSUE THREAD

**Issue by Nizami20052022** [vector-im/element-android#7508]
### Have you considered any alternatives?

Option to take advantage of the system language.

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
Could you provide some more info? We're already supporting the system language unless there's not a translation available for it.

**Comment by Nizami20052022** [comment]
Element application has Turkish language support.I applied the Turkish language to my phone.I deleted the data of the Element application, but the application is not in Turkish, but in English.The application does not use the default system language.

**Comment by jmartinesp** [comment]
If I'm not mistaken, the application saves the language the device has when it's first booted and will keep using it even if you change the language of the device. 

However, if you delete the app data this setting should be removed and a new language should be picked the next time the app starts.

I just tried it with English -> Spanish -> English again:
1. Phone was in English, changed it to Spanish: the app still has the English texts in the UI, if I check on Settings > Preferences > Language I can still see the English language selected.
2. Clear data from the app, restart it: texts are now in Spanish. Log in, check Preferences: Spanish is now the selected language there.
3. Change device to English again, clear app data: the texts are now in English again.

**Comment by yoguut** [comment]
> If I'm not mistaken, the application saves the language the device has when it's first booted and will keep using it even if you change the language of the device.
> 
> However, if you delete the app data this setting should be removed and a new language should be picked the next time the app starts.
> 
> I just tried it with English -> Spanish -> English again:
> 
> 1. Phone was in English, changed it to Spanish: the app still has the English texts in the UI, if I check on Settings > Preferences > Language I can still see the English language selected.
> 2. Clear data from the app, restart it: texts are now in Spanish. Log in, check Preferences: Spanish is now the selected language there.
> 3. Change device to English again, clear app data: the texts are now in English again.

Yes, I think you hit the problem there with your 3 steps to reproduce. 

The app doesn't dynamically observe the system's language. And as far as I can think of, there are 2 ways to approach this issue:

1. The app needs to observe the change in the system's language, and responds to changes
2. On app launch, check the system's language and then update the app's lang accordingly

I might be able to help with the code fix, but first of all. since OP is vanished. Let's reassign assign this issue to a new owner so that the description can be filled in according to contributing guidline.

I'm not sure who to ping here on GH. I can go poke around in the chat to get some help from a maintainer.
